templates/docker_rootless.service.j2

```diff
@@ -9,18 +15,21 @@ Documentation=https://docs.docker.com/engine/security/rootless/
 Environment="DOCKER_HOST=unix:///run/user/{{ docker_user_info.uid }}/docker.sock"
 Environment="PATH={{ docker_user_info.home }}/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
 Environment="XDG_RUNTIME_DIR=/run/user/{{ docker_user_info.uid }}"
+{% if docker_expose_docker_api_via_tcp %}
+Environment="DOCKERD_ROOTLESS_ROOTLESSKIT_FLAGS=-p 0.0.0.0:2375:2375/tcp"
+{% endif %}
 {% if not docker_rootful %}
 {% if ansible_distribution == "Debian" and ansible_kernel >= "4.18" and ansible_kernel <= "5.11" %}
-ExecStart={{ docker_user_info.home }}/bin/dockerd-rootless.sh -s fuse-overlayfs
+ExecStart={{ docker_user_info.home }}/bin/dockerd-rootless.sh {{ tcp_connection_str }} -s fuse-overlayfs
 {% else %}
-ExecStart={{ docker_user_info.home }}/bin/dockerd-rootless.sh
+ExecStart={{ docker_user_info.home }}/bin/dockerd-rootless.sh {{ tcp_connection_str }}
 {% endif %}
 {% endif %}
 {% if docker_rootful %}
 {% if ansible_distribution == "Debian" and ansible_kernel >= "4.18" and ansible_kernel <= "5.11" %}
-ExecStart=/usr/bin/dockerd-rootless.sh -s fuse-overlayfs
+ExecStart=/usr/bin/dockerd-rootless.sh {{ tcp_connection_str }} -s fuse-overlayfs
```

Comment:
I only tested with `docker_rootful = false` in my environment, not sure if the parameter stays the same in a `docker_rootful = true` scenario.